complearn-genbank/routes/redis.js

```diff
@@ -0,0 +1,66 @@
+const redis = require('redis');
+const express = require('express');
+const {error, info} = require("../configurations/logger");
+
+const router = express.Router();
+const client = redis.createClient({
+    url: process.env.REDIS_URL || 'redis://localhost:6379',
```

Comment:
centralize configuration read in envLoader.js, then call the same way as ENV_LOADER.GOOGLE_CLIENT_ID